scripts/Interface/ui/sans_isis/sans_data_processor_gui.py

```diff
@@ -249,6 +250,9 @@ def __init__(self):
 
         self._setup_progress_bar()
 
+        # At a later date we can drop new once we confirm the old GUI is not using "ISIS SANS"
+        UsageService.registerFeatureUsage("Interface", "ISIS SANS (new)", False)
```

Comment:
I searched through `qt/scientific_interfaces/ISISSANS`, and `git grep -i "registerFeatureUsage.*SANS"`, with no results. I'd say it's safe to drop the `(new)`, but you could check with someone that has usage reporting access and see if anything SANS shows up.